packages/calypso-build/webpack/sass.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+const MiniCssExtractPluginWithRTL = require( 'mini-css-extract-plugin-with-rtl' );
+
+module.exports.loader = ( { preserveCssCustomProperties, includePaths, prelude } ) => ( {
```

Comment:
JSDocs with types would be nice and can help some IDEs typecheck

Reply:
ba619de7d1